hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -3704,6 +3721,7 @@
             <backup-count>1</backup-count>
             <scale-up-delay-millis>10000</scale-up-delay-millis>
             <lossless-restart-enabled>false</lossless-restart-enabled>
+            <max-processor-accumulated-records>1000000000</max-processor-accumulated-records>
```

Comment:
This is quite a lot... Why not putting 1M?

Reply:
We want to provide an example and keep backward compatibility (not having it at all), I think 1M can be quite easily reached - I would say 1G is a fair compromise.